index.d.ts

```diff
@@ -189,3 +189,5 @@ export interface NodeInfoCommon<T extends NodeType> {
 export interface CallbackObject {
   build(): Promise<void>;
 }
+
+export type InputNodes = Node | string;
```

Comment:
Name it `InputNode`, it isn't plural there is still only one it is either a Node or a string.

Reply:
OOPS

Reply:
for example in the plugin you might have `inputNodes: InputNode[]`